xla/hlo/analysis/while_loop_analysis.cc

```diff
@@ -47,13 +49,12 @@ static const HloInstruction* NonConstantOperand(const HloInstruction* instr) {
   const HloInstruction* result = nullptr;
   for (const HloInstruction* operand : instr->operands()) {
     if (!operand->IsConstant()) {
-      if (result != nullptr) {
-        CHECK_EQ(result, operand);
```

Comment:
Please also update the comment in line 47

Reply:
Done